minecode/tests/testfiles/debian/lslr/ls-lR_debian

```diff
@@ -0,0 +1,87 @@
+.:
```

Comment:
Can you replace the `ls-lR_debian` and `ls-lR_ubuntu` files with the original ones from https://github.com/nexB/purldb/tree/main/minecode/tests/testfiles/debian/lslr, extracted from the .gz files? We want to use the same test data as before to see if we get the same expected results to ensure the fix that you created does not introduce any changes to how the function worked before. If you want, you can create an additional test with your new test ls-lR data.

Reply:
@JonoYang I changed the files to the original ones but, i had to make changes to the expected.json files as there were a few errors.
Example: 
all the purls in https://github.com/nexB/purldb/blob/main/minecode/tests/testfiles/debian/lslr/ls-lR_debian.gz-expected.json, are in wrong format ex: `pkg:deb/debian/asterisk-mobile@1.6.2.1-1?arch=amd64.deb`.

Reply:
@35C4n0r The idea is to use the previous ls-lR files and then update the expected JSON results.